### ISSUE TITLE
Transcribe and summarize videos in other languages

### ISSUE THREAD

**Issue by bkitapro** [logancyang/obsidian-copilot#972]
Copilot version:
Copilot Settings v2.7.11

**Describe how to reproduce**
Unfortunately copilot plus doesn't work for transcribing videos that aren't in English.

**Expected behavior**
Transcribe and summarize videos in other languages

**Screenshots**
https://imgur.com/a/pQn0Fip

**Comment by logancyang** [comment]
Thanks for reporting this, and sorry for the slow reply.

Copilot v3 is in code freeze, so we're closing this as not planned. v4 is a ground-up rewrite and is launching soon, with a very different architecture around agent mode.

If this is still an issue for you in v4 once it lands, please open a new issue and we'll take a fresh look. 🙏
